src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -507,7 +507,7 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
                   <button className='ms-PeoplePicker-resultAction' onClick={ () => {
                       this._removeSelectedPersona( selectedPersonas.indexOf(child) );
                   }}>
-                    <i className='ms-Icon ms-Icon--Cancel'></i>
+                    <i className='ms-Icon ms-Icon--Clear'></i>
```

Comment:
hey @Jahnp @atneik, pretty sure this is an error on my part.  I think I was working in another file that had --Clear instead of --Cancel, and mistaken made that change in this file as well.  I'll reverse the change in my next commit along with the RTL fixes.